public/app/project-setup/theme/mixins/_section.scss

```diff
@@ -2,9 +2,9 @@
 // $type - The type of the section: 'dialog' for dialogs. Default: false.
 @mixin section($type: false) {
   @if $type == dialog {
-    padding: $column-width 0 $column-width 0;
+    padding: $global-padding 0 $global-padding 0;
   }
   @else {
-    padding: $column-width;
+    padding: $global-padding;
```

Comment:
A global padding could be applied globally. This however is not applied globally. I'm not satisfied with the name.